editor/editor_property_name_processor.cpp

```diff
@@ -117,17 +117,23 @@ EditorPropertyNameProcessor::EditorPropertyNameProcessor() {
 	capitalize_string_remaps["bvh"] = "BVH";
 	capitalize_string_remaps["ca"] = "CA";
 	capitalize_string_remaps["cd"] = "CD";
+	capitalize_string_remaps["commentfocus"] = "Comment Focus";
```

Comment:
I don't think it's a good idea to add spaces to words that don't have any underscores in them. If I saw "Comment Focus" in the inspector, my inclination would be to type `comment_focus` into a script because in every other property, a space indicates an underscore. That could be confusing for users, so it would be better to rename these properties to include an underscore where needed instead of "fake" renaming them here.

This goes for "commentfocus", "defaultfocus", "default_frame", "hseparation", "selectedframe", "vadjust", and "vseparation"

Reply:
I'm not sure. We decided to remap things like `vsync` to `V-Sync` and `po2` to `Power of 2` at the very beginning.

The real property path is visible in tooltip and copyable via context menu or dragging to script directly. IMHO, the capitalization process is more like doing extra English localization from the property-name language. It's not guaranteed to be able to convert back.

And it's not always able to convert back even without the remapping process: e.g. both `http_2` and `http2` are `Http 2` after naive capitalization.

Reply:
I would change property paths in 4.0 to be more explicit and avoid some of the hacks added here.

But for now the baseline for localisation is `3.x` and we won't rename there, so it's better to keep things in sync for now